### ISSUE TITLE
startup error

### ISSUE THREAD

**Issue by Mischmaschine** [RediCloud/cloud-v1#93]
### Console Error

```
Failed to run node:
java.io.FileNotFoundException: Proxy plugin jar not found!
        at net.suqatri.redicloud.node.NodeLauncher.createDefaultFiles(NodeLauncher.java:517)
        at net.suqatri.redicloud.node.NodeLauncher.init(NodeLauncher.java:277)
        at net.suqatri.redicloud.node.NodeLauncher.<init>(NodeLauncher.java:94)
        at net.suqatri.redicloud.node.NodeLauncherMain.main(NodeLauncherMain.java:26)


```

### Bug Description

If you want to start the cloud, an error occurs 

### Steps to Reproduce

Start the cloud - you'll see the error instantly 

### Expected Behavior

normal cloud start

### Software/Hardware

OS: Ubuntu 22.04.1

### Cloud-Version

-

**Comment by obfuscaten** [comment]
Wenn man die Redi-Cloud herunterlädt, ist die alte Jar noch in der Zip. (redicloud-plugin-proxy -> redicloud-plugin-bungeecord)

**Comment by paulkczw** [comment]
Fixed [#3f6f6a0](https://github.com/RediCloud/cloud/commit/d5931e1a42c28d148cc0ee4252d0299c2bdb8beb)
